styles.css

```diff
@@ -0,0 +1,172 @@
+body {
+  margin: 0;
+  padding-left: 24px;
+  padding-right: 24px;
```

Comment:
Line 3 and 4 can best be replaced as padding: 0 24px;
Don't Repeat Yourself (D.R.Y)